instrumentation/othttp/handler.go

```diff
@@ -121,6 +148,15 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	setAfterServeAttributes(span, bw.read, rww.written, rww.statusCode, bw.err, rww.err)
 	span.SetStatus(standard.SpanStatusFromHTTPStatusCode(rww.statusCode))
+
+	// Add request metrics
+
+	labels := []kv.KeyValue{kv.Int("status", rww.statusCode), kv.String("method", r.Method)}
```

Comment:
Does it make sense to include any of the values that are added as trace attributes as labels here?  I'm thinking perhaps span name, hostname, TLS status, HTTP flavor, etc.

Reply:
+

Reply:
Yes, https://github.com/open-telemetry/opentelemetry-specification/blob/4a8ee03e7dd153968c854ebd6e9e690381a80417/specification/trace/semantic_conventions/http.md#http-server may be worthwhile reading here. I wonder about proliferation of labels though for some downstream processing/storage systems?

Reply:
Added standard labels

@jmacd pls review, cause I did some changes in standard package